spec/models/action_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'rails_helper'
+
+RSpec.describe Action, type: :model do
+  before(:each) do
+    @appointment = Appointment.new
+    @appointment.age = 42
+    @appointment.sex = "male"
+  end
+
+  it "after create an appointment with disease. Should create action automatically" do
+    expect(Action.all.count == 0)
+
+    hiv = Disease.create([{ name: "HIV" }])
+    @appointment.diseases << hiv
+    @appointment.save
+
+    expect(Action.all.count == 1)
```

Comment:
Eu acho que essa linha não faz o que você esperava.
Alterei para `expect(Action.all.count == 10)`, para confirmar, e o teste continuou passando.
Mas se eu mudo para `expect(Action.all.count).to eq(10)` aí sim o teste falha.

Reply:
Corrigido.